orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -222,7 +222,8 @@ class OperationsController {
         type: it.type,
         waitForCompletion: it.waitForCompletion,
         preconfiguredProperties: it.preconfiguredProperties,
-        noUserConfigurableFields: it.noUserConfigurableFields()
+        noUserConfigurableFields: it.noUserConfigurableFields(),
+        webhookParams: it.webhookParams
```

Comment:
maybe just `parameters` vs `webhookParams`

Reply:
exactly the type of feedback i wanted. i definitely like that better. thanks @ajordens!